scripts/cmake/vcpkg_build_make.cmake

```
@@ -148,7 +148,7 @@ function(vcpkg_build_make)
                 set(LINKER_FLAGS_${cmake_buildtype} "${VCPKG_DETECTED_CMAKE_STATIC_LINKER_FLAGS_${cmake_buildtype}}")
             else() # dynamic
                 set(LINKER_FLAGS_${cmake_buildtype} "${VCPKG_DETECTED_CMAKE_SHARED_LINKER_FLAGS_${cmake_buildtype}}")
-            endif()            
+            endif()

```

```suggestion
            endif()            
```

WS change. Will be removed after team review